hydra-node/test/Hydra/API/HTTPServerSpec.hs

```diff
@@ -147,6 +152,19 @@ apiServerSpec = do
                         (key "channels" . key "/snapshot/utxo" . key "subscribe" . key "message" . key "payload")
                   }
 
+      prop "has inlineDatumRaw" $ \i ->
+        forAll genTxOut $ \o -> do
+          let o' = modifyTxOutDatum (const $ mkTxOutDatumInline (123 :: Integer)) o
+          let getUTxO = pure $ Just $ UTxO.fromPairs [(i, o')]
+          withApplication (httpApp @Tx nullTracer dummyChainHandle defaultPParams cantCommit getUTxO getPendingDeposits putClientInput) $ do
+            get "/snapshot/utxo"
+              `shouldRespondWith` 200
+                { matchBody = MatchBody $ \_ body ->
+                    if isNothing (body ^? key (fromString $ show i) . key "inlineDatumRaw")
+                      then Just $ "\ninlineDatumRaw not found in body:\n" <> show body
```

Comment:
can we check here to verify the inlineDatumRaw value is actually 123?

Reply:
I realize it's a redundant check, but it might simplify the isNothing logic, which I initially found a bit confusing.

wdyt?